map/index.d.ts

```diff
@@ -37,7 +38,7 @@ export type MapStoreKeys<SomeStore> = SomeStore extends {
   : AllKeys<StoreValue<SomeStore>>
 
 export interface MapStore<Value extends object = any>
-  extends WritableAtom<Value> {
+  extends PreinitializedWritableAtom<Value> {
```

Comment:
No, we can’t add it to `MapStore` because this type can be used by someone else.

Let’s add `PreinitializedMap`.

Reply:
Added `PreinitializedMapStore`